consensus/polybft/test_utils.go

```diff
@@ -0,0 +1,33 @@
+package polybft
```

Comment:
Please rename this file to to include the `_test` prefix, as it doesn't need to be compiled into the final binary

cc @vcastellm

Reply:
Fixed through #825